webui/react/src/e2e/models/ant/Popover.ts

```diff
@@ -0,0 +1,49 @@
+import { BaseComponent, ComponentBasics } from 'e2e/models/BaseComponent';
+import { BasePage } from 'e2e/models/BasePage';
+
+interface requiresRoot {
+  root: BasePage;
+}
+
+interface PopoverArgsWithoutChildNode extends requiresRoot {
+  childNode?: never;
+  openMethod: () => Promise<void>;
+}
+
+interface PopoverArgsWithChildNode extends requiresRoot {
```

Comment:
These are exactly the same as the dropdown args interfaces. Can/should we combine them? `ClickableMenuArgs` or something maybe?

Reply:
Yeah a lot of this code is reused. I almost wanted them to be shared code, but it makes the super call a little harder. I'm not sure the best way to go about it, but you're right that I could maybe start with these args.

In addition, I'm not sure how ant works under the hood. If popovers and dropdowns are implemented separately, then I'd want mine to have separate implementation as well.

Reply:
https://github.com/ant-design/ant-design/blob/master/components/dropdown/dropdown.tsx
and i dont see any imports from `popover` in this file